pycloudlib/ibm/instance.py

```diff
@@ -681,7 +681,7 @@ def _attach_floating_ip_until_success(
                 )
         self._log.info(
             "Successfully attached floating ip: %s",
-            attached_floating_ip["name"],
```

Comment:
Got it; attached floating_ip is a bool. +1. and use_existing_floating_ip_w_name renamed.